CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to Cot
+
+Cot welcomes contribution from everyone in the form of suggestions, bug
+reports, pull requests, and feedback. This document gives some guidance if you
+are thinking of helping us.
+
+## Questions, feedback, and feature requests
+
+If you have a question, feedback, or feature request, please open an
```

Comment:
It might be worth adding issue templates

Reply:
Craated a task for that for now: https://github.com/orgs/cot-rs/projects/2?pane=issue&itemId=98048235